Makefile.in

```diff
@@ -493,4 +493,4 @@ dist: distclean
 	rm -f nagios-$(VERSION)
 
 # Targets that always get built
-.PHONY:	indent clean clean distclean dox test html
+.PHONY:	indent clean distclean dox test html
```

Comment:
sometimes duplicates can be intentional (there's some autoconf stuff that has 2 or 5 instances of `false` in a row...). My (admittedly rusty) recollection of make doesn't indicate this to be one of those times...